app/services/PageBuilder.scala

```diff
@@ -54,11 +54,11 @@ class PageBuilder extends ProcessPopulation {
     collectStanzas(key, Nil, Nil) match {
       case Right((ks, next, linked)) =>
         ks.head.stanza match {
-          case p: PageStanza if p.url.trim.isEmpty => Left(PageStanzaMissingOrUrlEmpty(ks.head.key))
+          case p: PageStanza if p.url.isEmpty || p.url.equals("/") => Left(PageUrlEmptyOrInvalid(ks.head.key))
```

Comment:
You might want to use trim in the right hand test (p.url.trim.equals("/"). Actually you may need trim on the left too. From REPL "   ".isEmpty = false

Reply:
Ah, I'd added trim within PageStanza when ensuring the leading /, i.e. the code which I removed. WIll add back